governance/votes/templates/Executive vote - September 13, 2024.md

```diff
@@ -0,0 +1,246 @@
+- UsdsJoin: [0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB](https://etherscan.io/address/0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB)
+- DaiUsds: [0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A](https://etherscan.io/address/0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A)
+
+#### sUSDS
+
+**Initialize sUSDS by calling [sUsdsInit.Init](https://github.com/makerdao/sdai/blob/susds/deploy/SUsdsInit.sol#L48) with the following parameters:**
+- susds: [0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD](https://etherscan.io/address/0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD)
+- sUsdsImp: [0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0](https://etherscan.io/address/0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0)
+- UsdsJoin: [0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB](https://etherscan.io/address/0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB)
+- usds: [0xdC035D45d973E3EC169d2276DDab16f1e407384F](https://etherscan.io/address/0xdC035D45d973E3EC169d2276DDab16f1e407384F)
+- [ssr](https://sky-atlas.powerhouse.io/#A.3.2.2.3_Sky_Savings_Rate-73d91126-21c9-4e8a-bedf-8a51a432bb59|57eab733e951): 6.25%
+
+#### SKY
+
+**Initialize SKY by calling [SkyInit.Init](https://github.com/makerdao/sky/blob/dev/deploy/SkyInit.sol#L41) with the following parameters:**
+- sky [0x56072C95FAA701256059aa122697B133aDEd9279](https://etherscan.io/address/0x56072C95FAA701256059aa122697B133aDEd9279)
+- mkrSky [0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B](https://etherscan.io/address/0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B)
+- [rate](https://sky-atlas.powerhouse.io/#A.4.1.2.1_SKY_Launch-d5af8504-ddd6-416d-8429-897497b072dc|b341f4c0b834): 24,000
+ 
+ These functions will also add the following elements to the chainlog:
+
+- usds address, `0xdC035D45d973E3EC169d2276DDab16f1e407384F`, under the key: "USDS".
+- usdsImp address, `0x1923DfeE706A8E78157416C29cBCCFDe7cdF4102`, under the key: "USDS_IMP".
+- UsdsJoin address, `0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB`, under the key: "USDS_JOIN".
+- DaiUsds address, `0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A`, under the key: "DAI_USDS".
+- sUsds address, `0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD`, under the key: "SUSDS".
+- sUsdsImp address, `0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0`, under the key: "SUSDS_IMP".
+- sky address, `0x56072C95FAA701256059aa122697B133aDEd9279`, under the key: "SKY".
+- mkrSky address, `0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B`, under the key: "MKR_SKY".
+ 
+The following permissions will be granted by the functions mentioned above:
+
+- The usdsJoin Adapter will be authorized in the usds contract by calling `rely` through the UsdsInit.init function.
+- The sUSDS contract will be authorized to access the `vat` by calling `rely` through the SUsdsInit.init function.
+- The mkrSky contract will be authorized in the sky contract by calling `rely` through the SkyInit.init function.
+- The mkrSky contract will be authorized in the MkrAuthority contract by calling `rely` through the SkyInit.init function.
```

Comment:
**very minor nit, non-blocking:** 

this could go under the relevant token initialization subheadings to have a more logical flow to the document

Reply:
Thanks, ldr! However, we prefer to group them so that the reader understands all the actions are related to permissions